apps/smart-contracts/core/test/Collateral.test.ts

```diff
@@ -941,6 +943,102 @@ describe('=> Collateral', () => {
     })
   })
 
+  describe('# setName', () => {
+    const nonBlankString = 'Not Blank'
+    it('reverts if not owner', async () => {
+      expect(await collateral.owner()).to.not.eq(user.address)
+
+      await expect(collateral.connect(user).setName(nonBlankString)).to.revertedWith(
+        revertReason('Ownable: caller is not the owner')
+      )
+    })
+
+    it('sets to a non-blank string', async () => {
```

Comment:
```suggestion
    it('sets to a non-empty string', async () => {
```
`empty` might be a better choice as `blank` usually refers to a string with whitespaces. Similarly for other places in tests.